client/package.json

```diff
@@ -29,7 +29,7 @@
     "standard": "^14.3.4",
     "stickyfilljs": "^2.1.0",
     "url-loader": "^3.0.0",
-    "webpack": "^4.44.2",
+    "webpack": "^5.24.4",
```

Comment:
I'm a little bit wary of jumping to a new major version without doing some checks on if any JS features break. Could we go to 4.46 instead? That should update the elliptical dependabot warning.

Reply:
I did do some checks and it all looked fine but now you've said that I don't want egg on my face when something breaks so putting it to that suggested version ;)